tools/reproduce_comparison/compareLinux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+[ $# -lt 1 ] && echo Usage: $0 SBOMURL && exit 1
```

Comment:
Do we need a set -ev?

Reply:
> Do we need a set -ev?

With the construct in there it's exiting appropriately when the test is false so I do not believe so, but feel free to try and convince me otherwise.